ftpbench/ftpbench.py

```diff
@@ -382,7 +398,7 @@ def main():
         opts = dict()
 
         opts["host"] = arguments["--host"]
-        if resolver and "," not in opts["host"]:
+        if resolver and "," not in opts["host"] and not re.match(r"\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", opts["host"]):
```

Comment:
Sorry, I don't get it. Why changing this logic?
This mask wont work with ipv6 addresses.